controllers/main.js

```diff
@@ -58,6 +58,31 @@ module.exports.updatePlayData = async (req, res, next) => {
   next();
 };
 
+module.exports.renderResults = async (req, res, next) => {
+  const playSessionData = JSON.stringify(
```

Comment:
Why are you stringifying the results?

Reply:
the reason was when i was passing without stringifying and later accessing it in `dispalyResults.js` i ran into issues (i think the object got converted to a string and i didnt know how to convert it back to an object). `JSON.stringify` was a solution that actually worked for me 
![image](https://github.com/Leedukas/pixel-thing/assets/100046819/59dd9ed4-e527-4c6d-8f2a-64f726b871ba)
![image](https://github.com/Leedukas/pixel-thing/assets/100046819/1c3c62ae-a54c-4f86-9d62-1f50b411c017)